episodes/19-basic-gis-with-r-sf.Rmd

```diff
@@ -191,13 +193,13 @@ ggplot() +
 
 ```
  
-## Area
+## Calculating area and density of spatial features
 
-*Problem: there are many pre-war buildings and the buffers are large so the number of old buildings is not very meaningful. Let's compute the density of old buildings per buffer zone.*
+In our analysis, we have a large number of pre-war buildings, and the buffer zones we’re using are quite broad. As a result, the total count of old buildings within these zones doesn’t provide us with the most meaningful insight. To make our analysis more useful, we should calculate the density of pre-war buildings within each buffer zone. This will help us better understand how these buildings are distributed across the area, providing more relevant and actionable information for our project.
 
 ```{r area}
 single_buffer$area <- st_area(single_buffer) %>% 
-  unit::set_units(., km^2)
+  units::set_units(., km^2)
```

Comment:
Are you sure `units::` is needed? If I run the episode with `library(units)` and `set_units()` without specifying the namespace, it seems to be working. Maybe there is a conflict with something you have installed on your system; I see no issues with the lesson packages recorded in renv.

Reply:
Issue to be solved in https://github.com/carpentries-incubator/r-geospatial-urban/issues/67